src/metabase/server/middleware/security.clj

```diff
@@ -51,7 +51,7 @@
                                   ["'self'"
                                    "'unsafe-eval'" ; TODO - we keep working towards removing this entirely
                                    "https://maps.google.com"
-                                   "https://apis.google.com"
+                                   "https://accounts.google.com"
```

Comment:
Are the other google/gstatic/googleapis even needed anymore?

Reply:
@flamber will we worthwhile looking in detail. Perhaps in a follow-up PR?